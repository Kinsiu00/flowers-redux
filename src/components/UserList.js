import React, {Component} from 'react';
import User from './User.js'

class UserList extends Component {
    render() {
        return (
            <div>
                <h4>List of Users</h4>
                {
                this.props.userData.map((eachUser) => {
                    return(
                        <User key={eachUser.id} eachUser={eachUser} setCurrentUser={this.props.setCurrentUser}/>
                    )
                })
                }
            </div>
        )
    }
}

export default UserList