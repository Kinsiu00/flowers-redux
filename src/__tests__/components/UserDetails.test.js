import React from 'react';
import { shallow } from 'enzyme';

import UserList from '../../components/UserDetails';

describe('<UserDetails>', () => {
    test('renders UserDetails', () => {
        const wrapper = shallow(<UserList currentUser={[]}/>)

        expect(wrapper).toMatchSnapshot()
    })
})