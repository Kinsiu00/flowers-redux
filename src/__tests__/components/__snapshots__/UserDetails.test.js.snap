// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDetails> renders UserDetails 1`] = `
<div>
  <p>
    Name: 
     
  </p>
  <p>
    Age: 
  </p>
  <p>
    Location: 
  </p>
  <p />
</div>
`;
